### PR TITLE
Clarify typography of abstract int 0 and abstract float 0

### PR DESCRIPTION
**Pull request by dneto0**
Addresses @ben-clayton feedback on a previous PR: https://github.com/gpuweb/gpuweb/pull/4449/files#r1447358137

### CODE REVIEW

File: wgsl/index.bs [hunk 1]
```diff
@@ -12233,8 +12233,8 @@ The zero values are as follows:
 * The zero value for an *C*-column *R*-row matrix of type *T* is the matrix of those dimensions filled with the zero value for *T*.
 * The zero value for a [=constructible=] *N*-element array with element type *E* is an array of *N* elements of the zero value for *E*.
 * The zero value for a [=constructible=] structure type *S* is the structure value *S* with zero-valued members.
-* The zero value of an [=AbstractInt=] is 0.
-* The zero value of an [=AbstractFloat=] is 0.0.
+* The zero value of an [=AbstractInt=] is `0`
+* The zero value of an [=AbstractFloat=] is `0.0`
```
Comment (mehmetoguzderin) [inline wgsl/index.bs:12237]: Is the omission of trailing dots intentional? Thank you

**Review by alan-baker — COMMENTED** [review]
I'd rather just address this as part of #4451. This change makes the bullets more inconsistent than before.

File: wgsl/index.bs (hunk 1 above)
Comment (dneto0) [reply wgsl/index.bs:12237]: Yes.
I don't want readers to mistake this for part of the syntax of the token.

**Comment by dneto0** [comment]
Close in favour of the broader work of #4451

**Pull request closed without merging**